dashboard/modules/reporter/reporter_head.py

```diff
@@ -23,9 +25,26 @@
 from ray._private.usage.usage_constants import CLUSTER_METADATA_KEY
 from ray.autoscaler._private.commands import debug_status
 
+from ray.util.state.common import (
+    ListApiOptions,
+)
+from ray.dashboard.state_aggregator import StateAPIManager
+from ray.util.state.state_manager import (
+    StateDataSourceClient,
+)
+
 logger = logging.getLogger(__name__)
 routes = dashboard_optional_utils.ClassMethodRouteTable
 
+EMOJI_WARNING = "&#x26A0;&#xFE0F;"
+WARNING_FOR_MULTI_TASK_IN_A_WORKER = "Warning: This task is running in a worker process that is running multiple tasks. The information that follows may come from any of these tasks:"
```

Comment:
```suggestion
WARNING_FOR_MULTI_TASK_IN_A_WORKER = "Warning: This task is running in a worker process that is running multiple tasks. This can happen if you are profiling a task right as it finishes or if you are using the Async Actor pattern. The information that follows may come from any of these tasks:"
```

Reply:
@scottsun94, what do you think?

Reply:
it might also be the case for "threaded actors"